iroha/src/sumeragi.rs

```diff
@@ -31,59 +35,126 @@ trait Consensus {
 #[derive(Io, Decode, Encode, Debug, Clone)]
 pub enum Message {
     /// Is sent by leader to all validating peers, when a new block is created.
-    Created(SignedBlock),
+    CreatedBlock(SignedBlock),
```

Comment:
To read it better use `BlockCreated` - otherwise it looks like type of the block.